tests/features/fixtures/minimalapp/.gitignore

```diff
@@ -0,0 +1,13 @@
+*.iml
+.gradle
+/local.properties
+/.idea/caches
```

Comment:
`.idea` should be sufficient to ignore any additional files that intellij might generate in future